src/cba-list-new/cba-list-new.js

```diff
@@ -569,7 +595,13 @@ class List extends HTMLElement
       const classes = ["row"];
       if (selected)
         classes.push("highlight");
-      const row = html`<span class="${classes.join(" ")}" tabindex="${selected ? 0 : -1}" draggable="${this.drag}" contenteditable="${editable}" title="${text}">${text}</span>`;
+      const addSubitemsButton = html`
+      <div class="subitem__wrapper">
```

Comment:
Yes, while we are using this elem in different cases it would be better to give it a generic, global name, such as .control class name. Double _  comes from BEM methodology and specifies blocks element classes.

Reply:
We are not using BEM in the project if you think BEM can be beneficiary we can discuss that, but we will need eventually change all the styles to BEM.